javascript-typing-tutor/main.js

```diff
@@ -0,0 +1,51 @@
+var $characterList = document.querySelectorAll('span');
+var $accuracy = document.querySelector('p');
+var $replay = document.querySelector('.replay');
+var totalPresses = 0;
+var numOfMistakes = 0;
+var i = 0;
+$characterList[i].className = 'current-letter';
+
+function handleKeyPress(event) {
+  if (event.key === $characterList[i].textContent) {
+    $characterList[i].className = 'green';
+    i++;
+    totalPresses++;
+    if (i === $characterList.length) {
+      $accuracy.textContent = 'Accuracy: ' + Math.floor(((totalPresses - numOfMistakes) / totalPresses) * 100) + '%';
+      $accuracy.className = 'accuracy show';
+      $replay.className = 'replay show';
+    } else $characterList[i].className = 'current-letter';
+  } else {
+    $characterList[i].className = 'current-letter red';
+    numOfMistakes++;
+    totalPresses++;
```

Comment:
Notice how you are incrementing `totalPresses` in both your `if` and `else` block. This means that you really want to do that work no matter what. You can move that line above the conditional, and that will allow you to only write that line of code once.